docs/source/conf.py

```diff
@@ -14,7 +14,7 @@
 import sys
 
 sys.path.insert(0, os.path.abspath("../.."))
-import cognite.replicator  # isort:skip
+import cognite.replicator.__version__  # isort:skip
```

Comment:
Could we achieve the same with:
import cognite.replicator.__version__ as version?

Reply:
Probably not